src/main.ts

```diff
@@ -192,9 +215,25 @@ export default class VaultChat extends Plugin {
 			}
 			return {
 				name,
-				contents
+				contents,
+				abstractFile
 			}
 		}))
 		return results.filter(isSearchResult)
 	}
+
+	showNoteChatView() {
+		this.app.workspace.detachLeavesOfType(NOTE_CHAT_VIEW);
+		this.app.workspace.getRightLeaf(false).setViewState({
+			type: NOTE_CHAT_VIEW,
+			active: true,
+		}).then(() => {
```

Comment:
What is this doing here?

Reply:
This is how you display the view per the docs: https://marcus.se.net/obsidian-plugin-docs/user-interface/views